source/MapPanel.cpp

```diff
@@ -1016,7 +1016,7 @@ void MapPanel::DrawTravelPlan()
 	bool hasEscort = false;
 	map<const Ship *, double> fuel;
 	for(const shared_ptr<Ship> &it : player.Ships())
-		if(!it->IsParked() && !it->CanBeCarried() && it->GetSystem() == flagship->GetSystem())
+		if(!it->IsParked() && (!it->CanBeCarried() || &*it == flagship) && it->GetSystem() == flagship->GetSystem())
```

Comment:
```suggestion
		if(!it->IsParked() && (!it->CanBeCarried() || it.get() == flagship) && it->GetSystem() == flagship->GetSystem())
```
`std::shared_ptr` has the `get()` method which directly returns the contained pointer.

Reply:
Actually, I did that first.  But in this case, it.get() returns a Ship, not a Ship *, so you have to dereference it anyway, and &*it looked cleaner than &it->get().

Reply:
> Actually, I did that first. But in this case, it.get() returns a Ship, not a Ship *, so you have to dereference it anyway, and &*it looked cleaner than &it->get().

??? `get` return a `Ship *`? Are you sure you're not dereferencing it?
![image](https://github.com/endless-sky/endless-sky/assets/101683475/6cd40cf0-f750-4582-a08d-d59dd5c626a7)

Reply:
Darn it, you're correct.  Ah, it must have been late at night when I thought that through (because I could swear I tried just that and got an error compiling).